src/lib_context/context.ml

```diff
@@ -169,7 +169,7 @@ and context = {index : index; parents : Store.Commit.t list; tree : Store.tree}
 
 type t = context
 
-module type S = Tezos_context_sigs.Context.S
+module type S = Tezos_context_sigs.Context.MEM
```

Comment:
Why did you rename this? This not the signature of the read-only backend :-)

Reply:
I made `Tezos_context_sigs.Context.S` be the signature of the full lib_context implementation (which used to be `lib_context/context.mli`.

I can look for a better naming than `MEM` for the old `Tezos_context_sigs.Context.S`